contrib/docker/build.sh

```diff
@@ -9,8 +9,8 @@ set -e
 echo "creating debian image..."
 docker run --rm -it -v `pwd`/../..:/build ebusd-buildenv ./make_debian.sh
 export EBUSD_VERSION=`cat ../../VERSION`
-export EBUSD_ARCH=`docker version|grep -i "Arch[^:]*server"|head -n 1|sed -e 's#^.*/##'`
+export EBUSD_ARCH=`docker version|grep -i "Arch:"|head -n 1|sed -e 's#^.*/##'`
 mv ../../ebusd-${EBUSD_VERSION}_${EBUSD_ARCH}_mqtt1.deb runtime/
 echo "building docker image..."
-(cd runtime && docker build -t ebusd .)
+(cd runtime && docker build -t ebusd --build-arg EBUSD_VERSION=${EBUSD_VERSION} --build-arg EBUSD_ARCH=${EBUSD_ARCH} .)
```

Comment:
my build machine does not allow using --build-arg as the docker in version 1.6.2 does not yet support it. that's why I was using ENV instead combined with the update.sh script.

Reply:
Oh, I see. I didn‘t even notice the update script. Then lets close this pr and cherry pick added git, removed apt cache plus better regex for update and build?

Reply:
okay good, hm, don't know how to cherry pick across repos. I'd just pick up your changes manually, or do you know a better way?

Reply:
No 😩 what I meant was manually if you want to...

Reply:
ok will do